src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js

```diff
@@ -9696,81 +9696,83 @@ oop.inherits(Mode, CMode);
 exports.Mode = Mode;
```

Comment:
@kevinushey @jjallaire thanks! I've ended up replacing only `mode/sql` with `ace/mode/sql` and reverting changes to `ace_uncompressed.js`. Highlighting looks good still in server and desktop.